chapters/Chapter 2 — Dressara.md

```diff
@@ -0,0 +1,94 @@
+The women of Dressara often wore dresses, which Leanne both admired and found a shame. Leanne enjoyed wearing dresses, but found women in suits much more handsome.
+
+Delphine was nothing if not handsome. She was tall, and did her best to look so: her fine Elisian suit lengthened her torso, and she had paired it, rather than with pants, with a skirt just long enough to wave gently in the light breeze the Dressarans were so fond of.
+
+Dressara was a large moon. It was somewhat pleasant, if you enjoyed their sort of pomp, unlike the dour planet Seven around which it held a lazy orbit. Like anywhere else, it was rather difficult to land on illegally, particularly when it was in the process of adjusting its orbit. 
+
+It was simpler to land legally. While Leanne didn’t have an Alt to identify her, nor to pilot her ship, she did have a collection of bounty cards for people on all the various planets and moons. Their targets were now long dead, but so long as the bodies weren’t found, Leanne could use their bounties to land (bounties did not expire).
+
+There only a few thousand inhabitants on Dressara, plus their few thousand servants. There was a party nearly every day, and everyone was invited—sometimes the servants, too. The men, Leanne knew, did little more than stand about all day, whether at parties or at home. They were utterly boring, both as people and as targets. The women, Leanne, assumed, did something more than simply wear dresses and stroll about—even the wealthy had to bring in money somehow—but whatever they did, it wasn’t interesting enough for Leanne to learn what it was. 
+
+But if any Dressaran woman—or, for that matter, any Dressaran at all—were ever to do anything interesting, Leanne would bet it would be Delphine. 
+
+Leanne looked away as Delphine approached. Her neck muscles tensed, her back straightened, her eyes glanced down—
+
+She jumped as a hand tapped her bare shoulder.
```

Comment:
- Leanne jumps, and then registers the hand tapping her shoulder. This is the more subjective point of view, and is what I have now. If going subjective, it would be nice if it could be even more so, with shorter sentences, etc.
- A hand taps her shoulder, and she jumps. This is the more objective point of view. 

This story has writing somewhere between objective and subjective. The writing style is definitely colored by Leanne's thoughts, and we don't tend to know much more than she does at any given time. However, we are also not limited exclusively to things she might think (this is not her inner monologue).

All that being said, I don't have better wordings right now.